.gitignore

```diff
@@ -1,4 +1,5 @@
 _state*
 .idea*
 .DS_Store
-bootstrap/bootstrap.go
\ No newline at end of file
+bin/*
+bootstrap/bootstrap.go
```

Comment:
Can we remove bootstrap.go please?

See #51 for context but TLDR:

We need bootstrap.go in so that `go get` works (makes sense). We need to come up with another approach because I really hate the idea of having a generated file in the repo. But for now please remove? Thoughts?